Unsorted/24292844_172adec.txt

```diff
@@ -20,4 +25,9 @@
 		お主も、釣る側になるよう<end_line>
 		努力せぇ<end_line>
 	</sjis>
+	<ascii>
+		One must strive to be the fisher<end_line>
+		you should as well,<end_line>
+		do your best not be a fish<end_line>
```

Comment:
Line 29 should be the end of a sentence

30/31 is kind of stiff. Especially with the repeated usage of the word fish. Since this is a fishing analogy, we can use something like "Do you best to avoid getting caught/hooked/snagged". Keeps the theme while changing up the wording to not be so repetitive.

Reply:
I changed it to:
> One must strive to be the fisher.<end_line>
> You too should do your best<end_line>
> to avoid getting hooked.<end_line>

commit: 64895e7